lib/phoenix_live_view/html_algebra.ex

```diff
@@ -194,15 +194,18 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
     doc =
       case lines do
         [] ->
-          empty()
+          line()
```

Comment:
Leaving this as `empty()` would require changing 1 test, but would effect all users formatted code so I elected to swap this to `line()`.

https://github.com/phoenixframework/phoenix_live_view/blob/d7c24e8e3bf17fe5ffe1abfceddd206f065440f3/test/phoenix_live_view/html_formatter_test.exs#L1411-L1421

The tests would be changed to `<script></script>` (and style).